docs/quickstart/google-cloud-platform.md

```diff
@@ -0,0 +1,114 @@
+
+<img src="../images/quickstart/google-cloud-platform/gcp2.png">
+
+Scroll down to the bottom and click `Create` to create and deploy the VM.
+
+Congrats you’ve created your VM instance!
+
+## SSH-ing into the VM
+
+On the Compute Engine Dashboard, click on the VM for this project. Under `Details`, click `SSH` and select `Open in browser window`.
+
+<img src="../images/quickstart/google-cloud-platform/gcp3.png">
+
+This will give you a terminal into the VM that we will use to install Coder.
+
+## Install Coder
+
+In the terminal, run the following command 
+
+```sh
+curl -fsSL https://coder.com/install.sh | sh  
+```
+
+## Run Coder
+
+For this tutorial, we will run Coder as a `systemd` service. You can run Coder in [a multitude of different ways](https://coder.com/docs/coder-oss/latest/install).
+
+First, edit the `coder.env` file to enable `CODER_TUNNEL` by setting the value to true with the following command:
+
+``` sh
+sudo vim /etc/coder.d/coder.env
+``` 
+
+<img src="../images/quickstart/google-cloud-platform/gcp4.png">
+
+Exit vim and run the following command to start Coder as a system service:
+
+```sh
+sudo systemctl enable --now coder
+``` 
+
+The following command shows the Coder service's logs, including the Access URL. The Access URL will be used to access the Coder control plane.  
+
+```sh
+journalctl -u coder.service -b 
+``` 
+
+<img src="../images/quickstart/google-cloud-platform/gcp5.png">
+
+In this instance, Coder can be accessed at the URL  `https://fcca2f3bfc9d2e3bf1b9feb50e723448.pit-1.try.coder.app`. 
+
+Copy the URL and run the following command to create the workspace admin:
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts and be sure to save use email and password. This is your admin login. Now, you can now access Coder from your local machine by navigating to the `***.try.coder.app` URL and logging in with that same username and password. 
+
+## Creating and Uploading your First Template
+
+First, run `coder template init` to create your first template. You’ll be given a list of prefabricated templates. This tutorial shows you how to create a Linux based template on GCP. 
+
+<img src="../images/quickstart/google-cloud-platform/gcp6.png">
+
+Select the `Develop in Linux on Google Cloud`, then `cd ./gcp-linux`. 
+
+Run the following command: 
+
+```sh
+coder templates create
+```
+
+It will ask for your `project-id`, which you can find on the home page of your GCP Dashboard. 
+
+Given it’s your first time setting up Coder, it may give an error that will look like the following:
+
+<img src="../images/quickstart/google-cloud-platform/gcp7.png">
+
+In the error message will be a link. In this case, the URL is `https://console.developes.google.com/apis/api/iam.googles.com/overview:?project=1073148106645`. Copy the respective URL from your error message, and visit it via your browser. It may ask you to enable `Identity and Access Management (IAM) API`. 
```

Comment:
My error message didn't contain a link :/

<img width="1360" alt="Screen Shot 2022-10-17 at 3 20 10 PM" src="https://user-images.githubusercontent.com/22407953/196275293-8da291ae-4b2a-4d87-8df7-a39c6b6b084e.png">

Reply:
I discovered that we can set the service account scopes during VM creation. I also tried changing the scopes on creating but it didn't seem to work.

<img width="534" alt="Screen Shot 2022-10-17 at 3 23 36 PM" src="https://user-images.githubusercontent.com/22407953/196276177-15f1271d-2068-4922-bfe9-e51dd5affbfa.png">